common/src/main/resources/ts/models/FormElement/FormElement.ts

```diff
@@ -17,19 +16,27 @@ export abstract class FormElement implements Selectable {
         this.form_id = null;
         this.title = null;
         this.position = null;
-        this.nb_responses = 0;
         this.selected = null;
         this.label = null;
     }
 
+    build(data: IFormElementResponse) : FormElement {
+        this.id = data.id ? data.id : null;
+        this.form_id = data.formId ? data.formId : null;
+        this.title = data.title ? data.title : null;
+        this.position = data.position ? data.position : null;
+        this.selected = data.selected ? data.selected : false;
+        this.label = data.label ? data.label : null;
```

Comment:
```suggestion
        this.label = data.label;
```
ça reviens au même ça, non ?

Reply:
Totalement ! C'était plus pour garder l'habitude de faire le check pour éviter d'oublier pour d'autres variables un jour